e2e/wdio/core/tests/panel.e2e-spec.ts

```diff
@@ -0,0 +1,73 @@
+import { PanelPo } from '../pages/panel.po';
+import {
+    click, getAttributeByName, getElementArrayLength, getElementSize, getText, isElementDisplayed,
+    refreshPage, scrollIntoView, waitForElDisplayed
+} from '../../driver/wdio';
+
+describe('Panel test suite', function() {
+    const panelPage = new PanelPo();
+    const { toggleButton, panelExpandableButton, panelParagraphs, expandablePanel, compactPanelButton, panelTitle } = panelPage;
+
+    beforeAll(() => {
+        panelPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForElDisplayed(toggleButton);
+    }, 1);
+
+    it('should check orientation', () => {
+        panelPage.checkRtlSwitch();
+    });
+
+    it('should check Expandable Panel example', () => {
+        scrollIntoView(toggleButton);
+        click(toggleButton);
+        expect(getAttributeByName(expandablePanel, 'ng-reflect-expanded')).toBe('false');
```

Comment:
this 'ng-reflect-expanded' is one of those dev mod attributes which we shouldn't use. You can see from the netlify deploy that this attribute isn't there https://fundamental-ngx.netlify.app/#/core/panel 
please rework to not use dev mod attribute. (I think you'll need to change the selector to use the drop-down arrow button and get 'aria-expanded' attribute.